engines/scumm/dialogs.cpp

```diff
@@ -1042,12 +1043,41 @@ void LoomTownsDifficultyDialog::handleCommand(GUI::CommandSender *sender, uint32
 
 // Game options widgets
 
-// Normally this would be added as a static game settings widget, but I see no
-// way to get both the dynamic and the static one, so we have to duplicate it
-// here.
+void ScummOptionsContainerWidget::load() {
+	if (_enhancementsPopUp)
+		_enhancementsPopUp->setSelectedTag(ConfMan.getInt("enhancements", _domain));
+}
+
+bool ScummOptionsContainerWidget::save() {
+	if (_enhancementsPopUp)
+		ConfMan.setInt("enhancements", _enhancementsPopUp->getSelectedTag(), _domain);
+	return true;
+}
+
+void ScummOptionsContainerWidget::createEnhancementsWidget(GuiObject *boss, const Common::String &name) {
+	GUI::StaticTextWidget *text = new GUI::StaticTextWidget(boss, name + ".EnhancementsLabel", _("Enhancements:"));
```

Comment:
Please add here comment for the translators, on a line above something like

```C++
// I18N: Game enhancements pop-up list
```

Reply:
Thanks, I've made the suggested changes. (For the bit flags, I blame spending too much time at work with a programming language that doesn't have shift operators. :-)